README.md

```diff
@@ -28,15 +28,11 @@ $ make install
 $ /usr/local/lib/python2.7.9/bin/python -V
 Python 2.7.9
 ```
-- Install protobuff, as we need it as a dependency:
-```
-$ sudo apt-get install build-essential protobuf-compiler libprotobuf-dev
 
-```
 - Clone the repo in your machine and `cd` into it:
 ```
 $ cd ~
-$ git clone git@github.com:ekumenlabs/terminus.git
+$ git clone https://github.com/ekumenlabs/terminus.git
```

Comment:
Agreed it is easier if you just want to clone and test. But is a pain if you want to push changes. Can you leave both? Something like:

 - Clone the repo in your machine and `cd` into it:
$ git clone git@github.com:ekumenlabs/terminus.git
or
$ git clone https://github.com/ekumenlabs/terminus.git